saleor/static/dashboard-next/products/views/ProductVariantCreate.tsx

```diff
@@ -79,8 +79,12 @@ export const ProductVariant: React.StatelessComponent<ProductUpdateProps> = ({
                       variantCreate({
                         variables: {
                           attributes: formData.attributes,
-                          costPrice: formData.costPrice,
-                          priceOverride: formData.priceOverride,
+                          costPrice: formData.costPrice
```

Comment:
What was submitted before? Shouldn't there be a helper function to do it?

Reply:
An empty string. It seems that we need to pass either string (which cannot be empty) or null. The problem is that cannot solve this problem by using just TS types, unless we pass `--passthrough-custom-scalars` option to `apollo-codegen` and write constructor for all those custom types.